plugins/modules/user.py

```diff
@@ -141,6 +141,7 @@ def modify_user(module):
                 opts += "%s=%s " % (attr, val)
         if load_module_opts is not None:
             opts = load_module_opts + opts
+        opts = opts.lower()
```

Comment:
wouldn't this change affect other attributes?
let's say for the attribute `capabilities` which uses capitalized value

reference
(https://www.ibm.com/docs/en/aix/7.2?topic=c-chuser-command)

Reply:
ok I'll check for account_locked before setting opts to be lower

Reply:
i don't think it should just be for `account_locked` but for all parameters that takes boolean input.
you might want to check the value instead, check if they are a True, False, Yes, No, Always, Never
and convert them to lower case before adding to `opts`